app/assets/javascripts/_init-autocomplete.js

```diff
@@ -12,32 +21,62 @@ jQuery.fn.extend({
               country: el.country,
               lat: el.lat,
               lon: el.lon
-            }
-          }))
+            };
+          }));
         });
-    },
-      select: function (event, ui) {
+      },
+      select: function(event, ui) {
         // by default the change event is not triggered on hidden input fields
         // we need it to update the map instantly
-        $('#' + this.id.replace(/city/, 'lat')).val(ui.item.lat).trigger('change');
-        $('#' + this.id.replace(/city/, 'lon')).val(ui.item.lon).trigger('change');
+        var lat = $("#" + this.id.replace(/city/, "lat"));
+        var lon = $("#" + this.id.replace(/city/, "lon"));
+        lat.val(ui.item.lat).trigger("change");
+        lon.val(ui.item.lon).trigger("change");
+        $(this).parent().removeClass('has-error');
```

Comment:
We try to use double quotes in the project's JS

Reply:
Yes you're absolutely right, I will edit it.